packages/view/backend/sirius-components-view-emf/src/main/java/org/eclipse/sirius/components/view/emf/diagram/ViewDiagramConversionData.java

```diff
@@ -25,6 +26,7 @@
  */
 public record ViewDiagramConversionData(
         Map<org.eclipse.sirius.components.view.diagram.NodeDescription, NodeDescription> convertedNodes,
-        Map<org.eclipse.sirius.components.view.diagram.EdgeDescription, EdgeDescription> convertedEdges
+        Map<org.eclipse.sirius.components.view.diagram.EdgeDescription, EdgeDescription> convertedEdges,
+        Map<org.eclipse.sirius.components.view.diagram.DiagramElementDescription, IDiagramElementDescription> convertedElements
```

Comment:
What is the point of this third map? Why is that needed on top of the two other ones? This is just the same data as the other two maps.

Reply:
Done